legion/sdk/legion/sdk/clients/deployment.py

```diff
@@ -102,7 +101,7 @@ def to_json(self, with_status=False) -> typing.Dict[str, str]:
         return result
 
 
-class ModelDeploymentClient(RemoteEdiClient):
+class ModelDeploymentClient(legion.sdk.clients.edi.RemoteEdiClient):
```

Comment:
Please, remove the full import path, and below too.

Reply:
It's required because we have build_client function in each python's modules for EDI clients. And I cant import and declare same function.